docs/Deploying/FAQ.md

```diff
@@ -19,7 +19,7 @@
 - [Should I use multiple enroll secrets?](#should-i-use-multiple-enroll-secrets)
 - [How can enroll secrets be rotated?](#how-can-enroll-secrets-be-rotated)
 - [What API endpoints should I expose to the public internet?](#what-api-endpoints-should-i-expose-to-the-public-internet)
-
+- 
```

Comment:
```suggestion
- 
```
I think you may be missing the content for this bullet. I could be wrong. It might be a surprise bullet :)

Reply:
Nice catch! Fixed.